lib/omniauth/strategies/odoo_keycloak.rb

```diff
@@ -11,7 +11,7 @@ class OdooKeycloak < OmniAuth::Strategies::KeycloakOpenId
 
       info do
         {
-          nickname: odoo_info[:vat].presence || raw_info["preferred_username"],
+          nickname: odoo_info[:name].presence || raw_info["preferred_username"],
```

Comment:
```suggestion
          nickname: raw_info["preferred_username"],
```
Shoudn't the nickname always be the "prefered_username"?

Reply:
But the preferred_username is optional, as pointed by the [docs](https://www.keycloak.org/docs-api/latest/rest-api/index.html#AccessToken). Shouldn't we keep a fallback?

Also, it was added not a long time ago in #2 

https://github.com/openpoke/decidim-module-odoo/pull/2/files#diff-ed909ce83b9e8af48486c4433e3c1f0f521d30c90bc3d34ab186df4c9902d8e1